src/app/services/usuario.service.ts

```diff
@@ -13,21 +13,35 @@ export class UsuarioService {
   constructor(private http: HttpClient,
     private router: Router) { }
 
-  login( formData: any ) {
+  login( formData: any ): Promise<string> {
+      return new Promise<string>(resolve => {
+        this.http.post(`${base_url}/login`,formData).subscribe(data => {
+          const msg = data['msg']
+          if(msg === 'Login realizado con exito'){
+            localStorage.setItem('token', data['token'])
+            localStorage.setItem('usuarioId', data['usuarioId'])
+            localStorage.setItem('profesional', data['profesional'])
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ComfyStream/ComfyStream-Frontend/pullRequest?prid=7138940)
```suggestion
            localStorage.setItem("profesional", data['profesional'])
```